src/hotspot/os_cpu/aix_ppc/atomicAccess_aix_ppc.hpp

```diff
@@ -26,392 +26,8 @@
 #ifndef OS_CPU_AIX_PPC_ATOMICACCESS_AIX_PPC_HPP
 #define OS_CPU_AIX_PPC_ATOMICACCESS_AIX_PPC_HPP
```

Comment:
Maybe remove these guards? The ones in the new file should be sufficient. (The same for the other files.)

Reply:
As long as no one is adding platform_cpu specific functions removing should be fine.